suite-native/module-send-receive/src/components/ReceiveTextHint.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Button, VStack, Text } from '@suite-native/atoms';
+
+type ReceiveTextHintProps = {
+    onShowAddress(): void;
+};
+
+const hintWrapperStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.medium,
+    alignItems: 'center',
+    textAlign: 'center',
+}));
+
+const hintTextStyle = prepareNativeStyle(_ => ({
+    textAlign: 'center',
```

Comment:
nit: if it's just single style you can use prop `align="center"` but this is also fine.

Reply:
fixed c3142a603